### PR TITLE
Removed "android:supportsRtl"

### PR DESCRIPTION
**Pull request by alirezaafkar**
Removed "android:supportsRtl" attribute from manifest because of "Manifest merger failed"

adding tools:replace"android:supportsRtl" to my application element didn't fix the problem.

### CODE REVIEW

**Comment by alirezaafkar** [comment]
I signed it!

File: easypermissions/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,10 +1,3 @@
...
-    package="com.google.example.easypermissions">
-
-    <application
-        android:label="@string/app_name"
-        android:supportsRtl="true">
-
-    </application>
-
+<manifest package="com.google.example.easypermissions">
+    <application/>
```
Comment (samtstern) [inline easypermissions/src/main/AndroidManifest.xml]: This is no longer valid XML.  How about this:

```
<manifest package="com.google.example.easypermissions">
    <application 
            android:label="@string/app_name">
    </application>
</manifest>
```

**Comment by alirezaafkar** [comment]
@samtstern I think it was valid! but this is okay too :)

**Comment by samtstern** [comment]
Oops you're right it was valid!  So what we really want is your XML but without removing the xmlns from the Manifest tag and the label from the application tag.

**Comment by alirezaafkar** [comment]
@samtstern So will you merge it please ?

**Pull request merged**